### PR TITLE
[ZT] add DNS resolver decisions, update logpush

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/analytics/logs/logpush.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+2. Click **Connect a service**.
+3. Enter a **Job name**.
+4. From the drop-down menu, choose the dataset to export.
+5. Next, select the data fields you want to include in the log.
+6. In the **Advanced settings** card, choose the timestamp format you prefer, and whether you want to enable logs sampling.
+7. Click **Next**.
+8. Select the service you want to export your logs to.
+9. Next, follow the service-specific instructions on the Zero Trust UI to validate your destination.
+
+The setup of your logpush integration is now complete. Logpush will send updated logs every five minutes to your selected destination.
```
Comment (abracchi-tw) [inline content/cloudflare-one/analytics/logs/logpush.md]: I wonder whether `logpush` is a proper noun? do we always capitalize it?

File: content/cloudflare-one/policies/filtering/dns-policies/_index.md [hunk 2]
```diff
@@ -62,9 +62,9 @@ When choosing the Block action, toggle the **Display custom block page** setting
 
 Policies with Override actions allow you to respond to all DNS queries for a given domain to another destination. For example, you can provide a custom response IP of `1.2.3.4` for all queries to www.example.com with the following policy:
```
Comment (abracchi-tw) [inline content/cloudflare-one/policies/filtering/dns-policies/_index.md]:
```suggestion
Policies with Override actions allow you to respond to all DNS queries for a given domain to another destination. For example, you can provide a custom response IP of `1.2.3.4` for all queries to `www.example.com` with the following policy:
```

File: content/cloudflare-one/analytics/logs/logpush.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Navigate to **Logs > Logpush**.
+2. Click **Connect a service**.
+3. Enter a **Job name**.
+4. From the drop-down menu, choose the dataset to export.
+5. Next, select the data fields you want to include in the log.
+6. In the **Advanced settings** card, choose the timestamp format you prefer, and whether you want to enable logs sampling.
+7. Click **Next**.
+8. Select the service you want to export your logs to.
+9. Next, follow the service-specific instructions on the Zero Trust UI to validate your destination.
```
Comment (abracchi-tw) [inline content/cloudflare-one/analytics/logs/logpush.md]:
```suggestion
9. Next, follow the service-specific instructions on the Zero Trust dashboard to validate your destination.
```
for consistency

**Review by abracchi-tw — APPROVED** [review]

File: content/cloudflare-one/analytics/logs/logpush.md (hunk 3 above)
Comment (ranbel) [reply content/cloudflare-one/analytics/logs/logpush.md]: fixed in new commit

File: content/cloudflare-one/analytics/logs/logpush.md (hunk 1 above)
Comment (ranbel) [reply content/cloudflare-one/analytics/logs/logpush.md]: I believe so. It's capitalized in the logs documentation.

**Pull request merged**